src/hotspot/share/runtime/mutex.cpp

```diff
@@ -523,6 +533,61 @@ void Mutex::set_owner_implementation(Thread *new_owner) {
   }
 }
 #endif // ASSERT
+// Print all mutexes/monitors that are currently owned by a thread; called
```

Comment:
Please, leave a blank line between the pre-existing functions and the ones you are moving.

```suggestion

// Print all mutexes/monitors that are currently owned by a thread; called
```